config/templates/operator/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubearchive-operator-config
+  namespace: kubearchive
+  annotations:
+    # This annotation causes the operator deployment to restart when the ConfigMap changes
+    reloader.stakater.com/match: "true"
```

Comment:
This needs a separate controller: https://github.com/stakater/Reloader

Reply:
No, it's not being used.  I'll remove the comment.